docs/source/th/using-diffusers/controlling_generation.md

```diff
@@ -0,0 +1,197 @@
+
+แรกเราใช้การใส่ส่วนจำกัดของข้อมูลขาเข้าที่ต้องการให้มีผลต่อคำแนะนำที่ให้เป้าหมายเป็นค่าใหม่ที่ส่วนใหญ่เป็นศูนย์ จากนั้นนำผลลัพธ์ไปใช้ในการกระตุ้นค่าการควบคุมสำหรับส่วนของภาพที่ตรงกับข้อมูลขาเข้าที่ถูกแก้ไข ทำให้คำแนะนำมีผลต่อภาพที่สนใจ
+
+ดูเพิ่มเติม[ที่นี่](../api/pipelines/self_attention_guidance) เพื่อข้อมูลเพิ่มเติมเกี่ยวกับวิธีการใช้
+
+## Depth2Image
+
+[Paper](https://arxiv.org/abs/2302.03027)
+
+[Depth2Image](../api/pipelines/depth2image) ช่วยให้ผู้ตรวจสอบในคำแนะนำถูกแสดงในภาพที่สมบูรณ์
+
+เราสร้างคำแนะนำเพื่อให้ภาพที่มีความลึกเป็นผลลัพธ์ โดยรวมทั้งรูปภาพและคำแนะนำเพื่ออธิบายความลึก สำหรับกระบวนการลด noise  เราใช้การแก้ไขข้อมูลขาเข้าเพื่อเพิ่มความสำคัญของลำดับคำแนะนำที่ทำให้รูปภาพมีความลึก
+
+การนำเข้าภาพและคำแนะนำจะถูกทำให้เหมือนกับว่าคำแนะนำกำหนดความลึกของภาพ ซึ่งจะถูกใช้ในการกระตุ้นการแก้ไขข้อมูลขาเข้า โดยให้ความสำคัญสูงของลำดับคำแนะนำนั้น ๆ
+
+ดูเพิ่มเติม[ที่นี่](../api/pipelines/depth2image) เพื่อข้อมูลเพิ่มเติมเกี่ยวกับวิธีการใช้
+
+## MultiDiffusion Panorama
+
+[Paper](https://arxiv.org/abs/2302.03027)
+
+[MultiDiffusion Panorama](../api/pipelines/multidiffusion_panorama) ช่วยให้ผู้ตรวจสอบในคำแนะนำถูกแสดงในภาพที่สมบูรณ์
+
+กระบวนการลด noise ทำงานบนภาพที่มีขนาดใหญ่กว่า (เช่น รูปภาพทิ้งท้าย) โดยใช้หลายเส้นทางลด noise พร้อมกัน
+แต่ละเส้นทางลด noise ได้รับการควบคุมด้วยตัวแปรกลางขึ้นอยู่กับกระบวนการและตำแหน่งที่ให้คำแนะนำในภาพ
+
+ในการลด noise แต่ละครั้ง คำแนะนำทั้งหมดจะถูกใช้ในการกระตุ้นตัวแปรกลางและนำไปสู่ผลลัพธ์ของกระบวนการ
+ทำให้กระบวนการและตำแหน่งของคำแนะนำมีผลต่อภาพผลลัพธ์ที่ได้
+
+ดูเพิ่มเติม[ที่นี่](../api/pipelines/multidiffusion_panorama) เพื่อข้อมูลเพิ่มเติมเกี่ยวกับวิธีการใช้
+
+## DreamBooth
+
+[Paper](https://arxiv.org/abs/2302.03027)
+
+[DreamBooth](../api/pipelines/dreambooth) เป็นกระบวนการที่ช่วยในการแก้ไขภาพที่สามารถนำไปใช้ใน DreamBooth
+
+เราสร้างคำแนะนำที่เกี่ยวข้องกับการแก้ไขที่ควรทำกับภาพที่สมบูรณ์ใน DreamBooth
+ซึ่งเป็นกระบวนการที่ช่วยในการเพิ่มความเป็นฝันในรูปภาพ สำหรับกระบวนการลด noise  เราใช้การแก้ไขข้อมูลขาเข้าเพื่อเพิ่มความสำคัญของการใช้คำแนะนำที่เข้าข่ายในภาพ
+
+การนำเข้าภาพและคำแนะนำจะถูกให้เหมือนกับว่าคำแนะนำกำหนดการแก้ไขที่สร้างภาพที่เป็นฝัน ซึ่งจะถูกใช้ในการกระตุ้นการแก้ไขข้อมูลขาเข้า โดยให้ความสำคัญสูงของคำแนะนำที่เข้าข่าย
+
+ดูเพิ่มเติม[ที่นี่](../api/pipelines/dreambooth) เพื่อข้อมูลเพิ่มเติมเกี่ยวกับวิธีการใช้
+
+## Textual Inversion
+
+[Paper](https://arxiv.org/abs/2302.03027)
+
+[Textual Inversion](../api/pipelines/textual_inversion) ช่วยให้ผู้ตรวจสอบในคำแนะนำถูกแสดงในภาพที่สมบูรณ์
+
+การเริ่มต้นจากคำแนะนำที่เกี่ยวข้องกับภาพที่ต้องการแก้ไข เราสร้างคำแนะนำเพื่อให้ภาพผลลัพธ์เป็นภาพที่สนใจ สำหรับกระบวนการลด noise  เราใช้การแก้ไขข้อมูลขาเข้าเพื่อเพิ่มความสำคัญของการใช้คำแนะนำนั้น ๆ
+
+การนำเข้าภาพและคำแนะนำจะถูกให้เหมือนกับว่าคำแนะนำกำหนดภาพที่สนใจ ซึ่งจะถูกใช้ในการกระตุ้นการแก้ไขข้อมูลขาเข้า โดยให้ความสำคัญสูงของคำแนะนำ
+
+ดูเพิ่มเติม[ที่นี่](../api/pipelines/textual_inversion) เพื่อข้อมูลเพิ่มเติมเกี่ยวกับวิธีการใช้
+
+## Captcha
```

Comment:
I don't think we have a Captcha pipeline. It also looks like we're missing everything from ControlNet to Fabric.